script/generate-red-worker-code.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/**
+ * This script transpiles `RedundantAudioEncoder.ts` into JavaScript code and generates a file that stores the
+ * JavaScript code as a string which can be used to run a redundant audio worker. This script is meant to be run before
+ * building the entire project so that the generated file also gets built.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+// Create a temporary tsconfig file to only transpile `RedundantAudioEncoder.ts`.
+const configDir = 'config';
+const redTsconfig = 'tsconfig.red.json';
+const redTsconfigContent = `{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../build",
+    "rootDir": "../src"
+  },
+  "include": [
+    "../src/redundantaudioencoder/RedundantAudioEncoder.ts",
+  ]
+}
+`;
+fs.writeFileSync(`${configDir}/${redTsconfig}`, redTsconfigContent);
```

Comment:
could we add this temp file to .gitignore?

Reply:
sure, let me add it